src/store/modules/dashboard/type.ts

```diff
@@ -0,0 +1,36 @@
+
+export const VIEWERS_TYPE = Object.freeze({
```

Comment:
suggestion: I recommend not to use both `as const` and `Object.freeze()`

```suggestion
export const VIEWERS_TYPE = {
```

Reply:
Oh.. I missed it.